doc/build.sh

```diff
@@ -37,12 +37,7 @@ EOF
 done
     cat <<EOF >>overrides/partials/versions.html
         <option style="color:white;background-color:#4b2a5c;" value="archive">Archive</option>"
-EOF
-    cat <<EOF >>overrides/partials/versions.html
```

Comment:
@nadegepepin This was a lint error in CI. I think I've got the intended output, but please check the changes again to make sure it's doing the right thing for docs.